gp-includes/things/translation-set.php

```diff
@@ -472,6 +472,27 @@ public function update_status_breakdown() {
 					'public'             => (int) $warnings_counts->public,
 				);
 			}
+
+			$untranslated_counts = $wpdb->get_row( $wpdb->prepare( "
+				SELECT
+					COUNT(*) AS total
+				FROM {$wpdb->gp_originals} AS o
+				LEFT JOIN {$wpdb->gp_translations} AS t ON o.id = t.original_id AND t.translation_set_id = %d AND t.status != \"rejected\" AND t.status != \"old\"
+				WHERE
+					o.project_id = %d AND
+					o.status = \"+active\" AND
+					t.translation_0 IS NULL
+			", $this->id, $this->project_id ) );
+
+			if ( $untranslated_counts ) {
+				$counts[] = (object) array(
+					'translation_status' => 'untranslated',
+					'total'              => (int) $untranslated_counts->total,
+					'public'             => (int) $untranslated_counts->total,
+					'hidden'             => 0,
```

Comment:
I wasn't sure if you could have a hidden untranslated string, but it would easy to add.